### PR TITLE
story/TYTA-99/create-project-hierarchy

### PR DESCRIPTION
**Pull request by remirubis**
### Description

Updating the file hierarchy to create consistency within the application, we will then create an Api folder that will have all the methods calling the API.

Don't hesitate to give me your opinion about it 🥇

### CODE REVIEW

File: src/Router.jsx [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+      path: '/',
+      element: <DefaultLayout />,
+      children: [
+        { path: '', element: <Home /> },
+        // { path: 'register', element: <Register /> },
+        // { path: 'login', element: <Login /> },
+      ],
+    },
+    { path: '404', element: <h1>Not found</h1> },
+    { path: '*', element: <Navigate to="/404" replace /> },
```
Comment (BrystoQ) [inline src/Router.jsx]: Is there no cleaner way?

**Review by BrystoQ — DISMISSED** [review]
Juste maybe try to find another way to do 404 page, maybe create a layout in future

File: src/Router.jsx (hunk 1 above)
Comment (remirubis) [reply src/Router.jsx]: Yes in another ticket! But thanks 😄

File: src/components/Sections/Toolbar/Toolbar.jsx [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+                primary="Manage project"
+                primaryTypographyProps={{
+                  fontSize: '.9rem',
+                  fontWeight: 'medium',
+                  lineHeight: '20px',
+                  mb: '2px',
+                }}
+                sx={{ my: 0 }}
+              />
+            </ListItemButton>
```
Comment (FabienHP) [inline src/components/Sections/Toolbar/Toolbar.jsx:173]: Becare of Footer pass through the button

**Review by FabienHP — APPROVED** [review]

File: src/components/Sections/Toolbar/Toolbar.jsx (hunk 2 above)
Comment (remirubis) [reply src/components/Sections/Toolbar/Toolbar.jsx:173]: You made a comment but you merged it? Why not 😂

**Pull request merged**